src/internal/clusterstate/v2.8.0/pfsdb.go

```diff
@@ -249,6 +247,17 @@ func alterCommitsTable(ctx context.Context, tx *pachsql.Tx) error {
 	return nil
 }
 
+func alterCommitsTablePostDataMigration(ctx context.Context, env migrations.Env) error {
+	query := `
+	ALTER TABLE pfs.commits
+	    DROP CONSTRAINT fk_col_commit;
```

Comment:
What constraint is this?

Reply:
that constraint says that every item in pfs.commits needs to have a corresponding entry in collections.commits.

Reply:
But there isn't a `not null` constraint on `commit_id` so you can safely insert a commit with `commit_id = null`.

Reply:
but why would we want this constraint once the migration is complete?

Reply:
We can drop this constraint after the data migration though. This way we don't have to introduce a "temporary" migration state.